src/ImageSharp/Formats/Jpeg/JpegEncoderCore.cs

```diff
@@ -550,7 +550,8 @@ private void WriteApplicationHeader(ImageMetaData meta)
         private void WriteDefineHuffmanTables(int componentCount)
         {
             // Table identifiers.
-            byte[] headers = { 0x00, 0x10, 0x01, 0x11 };
+            Span<byte> headers = stackalloc byte[] { 0x00, 0x10, 0x01, 0x11 };
```

Comment:
We should keep an eye out for more things like this but I suspect you already have it covered 👍